app/Http/Controllers/UsersController.php

```diff
@@ -373,24 +374,46 @@ public function beatmapsets($_userId, $type)
      */
     public function index()
     {
-        $params = get_params(request()->all(), null, ['ids:int[]']);
+        $params = get_params(request()->all(), null, [
+            'ids:int[]',
+            'include_variant_statistics:bool',
+        ]);
 
         $includes = UserCompactTransformer::CARD_INCLUDES;
 
         if (isset($params['ids'])) {
             RequestCost::setCost(count($params['ids']));
+            $includeVariantStatistics = $params['include_variant_statistics'] ?? false;
```

Comment:
wouldn't that encourage people to request `/users/{user}` many times instead of using this endpoint? I don't know how to come up with good costs but they should at least incentivize making less requests for similar data

Reply:
this adds 2n queries all of which are potentially rather slow.

Reply:
I understand the cost increase but not so much how it compares to `/users/{user}/mania` (seemingly far more expensive than one user in `/users`, but again I don't know how to properly measure this)

anyway I guess this whole request cost system is in early stages so maybe not a big deal for this PR. I applied your suggestion and ig we'll just see if that's problematic for the people who use this?